magicgui/widgets/_function_gui.py

```diff
@@ -186,23 +187,23 @@ def __init__(
 
         if call_button is None:
             call_button = not auto_call
-        self._call_button: PushButton | None = None
+        self.call_button: PushButton | None = None
```

Comment:
actually, i might prefer a public accessor rather than changing the private attr.

```python
@property
def call_button(self) -> Optional[PushButton]:
    return self._call_button
```